StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -439,7 +438,7 @@ extension PaymentMethodFormViewController {
         }
         let additionalParameters: [String: Any] = [
             "product": "instant_debits",
-            "attach_required": true,
+//            "attach_required": true,
```

Comment:
Is this intentionally left commented out?

Reply:
Nope, that’s an oversight.